lib/changeNIDMtoxSPM.m

```diff
@@ -55,6 +55,7 @@
     excursionSetMaps = searchforType('nidm_ExcursionSetMap', graph);
     mipFilepath = searchforID(excursionSetMaps{1}.nidm_hasMaximumIntensityProjection.('x_id'),graph);
     nidmTemp.MIP = fullfile(json.filepath, mipFilepath.('prov_atLocation').('x_value'));
+    nidmTemp.oW = json.overWrite;
```

Comment:
Consider keeping `overWrite` rather than `oW` for readability. But, we might not need to keep this field (cf. this comment below).